avm/res/container-instance/container-group/tests/e2e/defaults/main.test.bicep

```diff
@@ -36,37 +36,39 @@ resource resourceGroup 'Microsoft.Resources/resourceGroups@2021-04-01' = {
 // ============== //
 
 @batchSize(1)
-module testDeployment '../../../main.bicep' = [for iteration in [ 'init', 'idem' ]: {
-  scope: resourceGroup
-  name: '${uniqueString(deployment().name, resourceLocation)}-test-${serviceShort}-${iteration}'
-  params: {
-    location: resourceLocation
-    name: '${namePrefix}${serviceShort}001'
-    containers: [
-      {
-        name: '${namePrefix}-az-aci-x-001'
-        properties: {
-          image: 'mcr.microsoft.com/azuredocs/aci-helloworld'
-          ports: [
-            {
-              port: 443
-              protocol: 'Tcp'
-            }
-          ]
-          resources: {
-            requests: {
-              cpu: 2
-              memoryInGB: 2
+module testDeployment '../../../main.bicep' = [
+  for iteration in ['init', 'idem']: {
```

Comment:
Question: Do you have the latest upstream version merged? I.e., does your readme generation still work?

Reply:
Yesterday I did the latest updates.

Reply:
Now only the static tests fail. If you want to unblock this PR you'd need to remove the line break in the test files (only there, not the rest of the templates) - OR - wait for PR #1368

Reply:
Thanks @AlexanderSehr for the update 🙂

Reply:
@AlexanderSehr its done :)